fluidsynth/src/synth/fluid_voice.h

```diff
@@ -143,15 +143,10 @@ int fluid_voice_set_output_rate(fluid_voice_t* voice, fluid_real_t value);
     function.*/
 void fluid_voice_update_param(fluid_voice_t* voice, int gen);
 
-/**  fluid_voice_release
- Force the voice into release stage. Usefuf anywhere a voice
- needs to be damped even if pedals (sustain sostenuto) are depressed.
- See fluid_synth_damp_voices_LOCAL(), fluid_synth_damp_voices_by_sostenuto_LOCAL,
- fluid_voice_noteoff(), fluid_synth_stop_LOCAL().
-*/
```

Comment:
A more up to date comment for this function is present at https://github.com/FluidSynth/fluidsynth/blob/80b46c601ad27ab3298f38e02e36ed602f1793a7/fluidsynth/src/synth/fluid_voice.c#L1186

Also, I'm afraid you are the only available reviewer currently, as the others are inactive (did you got my mail?).